File: Selenium/Activities/Activity19.java
package activities;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity19 {

	public static void main(String[] args) {
		
		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/selenium/javascript-alerts");
		
		System.out.println(driver.getTitle());
		
		driver.findElement(By.id("confirm")).click();

        Alert confirmAlert = driver.switchTo().alert();

        String alertText = confirmAlert.getText();
        System.out.println("Text in alert: " + alertText);

        confirmAlert.accept();
        
        //confirmAlert.dismiss();

        driver.close();
	}

}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://v1.training-support.net/selenium/javascript-alerts")

    print(driver.title)

    driver.find_element(By.ID, "confirm").click()
   
    confirm_alert = driver.switch_to.alert

    print(confirm_alert.text)

    confirm_alert.accept()
    
    #confirm_alert.dismiss()

    driver.close()